jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java

```diff
@@ -79,8 +84,18 @@ public void setFrom(Object from) {
      *
      * By default this is "from.getClass()". This takes
      * precedence over the {@link #setFrom(Object)} method.
+     *
+     * This used to be called {@code setClass}, but that ended up causing
+     * problems with new commons-beanutils restrictions via
+     * {@code ConvertingWrapDynaBean} use in {@code JellyBuilder}.
      */
+    public void setClazz(Class clazz) {
+        this.clazz = clazz;
+    }
+
+    @Deprecated // TODO Remove this method?
     public void setClass(Class clazz) {
+        LOGGER.log(Level.WARNING, "Unexpected call to #setClass", new Exception());
```

Comment:
FYI this warning might be a bit noisy. IIUC plugin maintainers won't be able to migrate to the new syntax until after they update their baseline to the first version of Jenkins core where this is released, since if they need to support older versions of Jenkins core they need to keep using the old syntax in their Jelly files. Besides, I found 1,070 unique instances of `<st:include class=""/>` attributes that needed updating. I doubt that log files and Jira tickets are going to help speed this migration along. Rather I wrote a quick and dirty tool at https://github.com/basil/jelly-migrator/ that can be used once a plugin's Jenkins core baseline is updated to automatically update all Jelly files to the new syntax.

Reply:
See https://github.com/jenkinsci/jenkins/pull/5324#issuecomment-790371595

Because of the migration code in `StaplerTagLibrary` I expect there to be absolutely no callers left.

Reply:
Added escape hatch that would allow admins to disable this warning.